modules/command-engine/core/src/main/scala/surge/internal/persistence/PersistentActor.scala

```diff
@@ -199,6 +199,8 @@ class PersistentActor[S, M, R, E](
     super.preStart()
   }
 
+  private def surgeContext() = Context(businessLogic.executionContext, self)
```

Comment:
How would you see a team interacting with the `ActorRef` here? There's a lot it's doing under the hood and it's very internal at the moment so my only concern would be that we could break consumers using the ActorRef relatively easily without knowing

Reply:
I anticipate that we will expose functions that interact with actor, but I do not intend to expose the actor ref directly to the app.

For instance, something like

```
def scheduleAlarm(time: Instance) = ScheduleAlarm(time) ? actorRef
```